### ISSUE TITLE
Sample buy

### ISSUE THREAD

**Issue by Aleandro21** [Lu-Yi-Hsun/iqoptionapi#11]
@Lu-Yi-Hsun
i have a error when 
# coding=utf-8
from iqoptionapi.stable_api import IQ_Option
import logging
import time
logging.basicConfig(level=logging.DEBUG,format='%(asctime)s %(message)s')
I_want_money=IQ_Option("email","pass")
goal="EURUSD"
print("get candles")
print(I_want_money.get_candles(goal,60,111,time.time()))
Money=1
ACTIVES="EURUSD"
ACTION="call"
expirations_mode=1
force_buy= True
I_want_money.buy(Money,ACTIVES,ACTION,expirations_mode,force_buy)

Why ?
TypeError: buy() takes exactly 4 arguments (6 given)

**Comment by Aleandro21** [comment]
if i change I_want_money.buy(1,'EURUSD','call')

i have this error
2018-05-07 10:05:15,162 {"msg": {"direction": "call", "price": 1, "act": "1", "exp": 1525680360, "time": 1525680313, "type": "turbo"}, "name": "buyV2"}
2018-05-07 10:05:15,190 {"name":"buyComplete","request_id":"","msg":{"isSuccessful":false,"message":["Invalid request"],"result":{}}}

**Comment by Lu-Yi-Hsun** [comment]
read  README.md  i have update new Features

**Comment by Lu-Yi-Hsun** [comment]
update last version , i can work

**Comment by Aleandro21** [comment]
@Lu-Yi-Hsun thanks, work !!
you have implemented a signal ?

**Comment by Lu-Yi-Hsun** [comment]
What signal?

**Comment by Aleandro21** [comment]
@Lu-Yi-Hsun same this code
https://github.com/harwee/IQOption-Api
for this code i have this error 

if self.api.timesync.server_datetime.second == 0:
File "/usr/local/lib/python2.7/dist-packages/iqoptionapi/ws/objects/timesync.py", line 37, in server_datetime
return datetime.datetime.fromtimestamp(self.server_timestamp)
File "/usr/local/lib/python2.7/dist-packages/iqoptionapi/ws/objects/timesync.py", line 24, in server_timestamp
return self__server_timestamp / 1000
NameError: global name 'self__server_timestamp' is not defined

**Comment by Lu-Yi-Hsun** [comment]
you mean analysis signal when to buy?

**Comment by Aleandro21** [comment]
@Lu-Yi-Hsun 
sorry the correct link is https://github.com/n1nj4z33/iqoptionbot

**Comment by Aleandro21** [comment]
@Lu-Yi-Hsun yes tis code i use for signal of buy or sell

**Comment by Lu-Yi-Hsun** [comment]
is that n1nj4z33/iqoptionbot work well?

**Comment by Aleandro21** [comment]
@Lu-Yi-Hsun i have a error : 
f self.api.timesync.server_datetime.second == 0:
File "/usr/local/lib/python2.7/dist-packages/iqoptionapi/ws/objects/timesync.py", line 37, in server_datetime
return datetime.datetime.fromtimestamp(self.server_timestamp)
File "/usr/local/lib/python2.7/dist-packages/iqoptionapi/ws/objects/timesync.py", line 24, in server_timestamp
return self__server_timestamp / 1000
NameError: global name 'self__server_timestamp' is not defined

**Comment by Aleandro21** [comment]
yes 1nj4z33/iqoptionbot analysis well , with some changes of mine

**Comment by Lu-Yi-Hsun** [comment]
ok i fork that try to fix

**Comment by Aleandro21** [comment]
@Lu-Yi-Hsun ok thanks ,  you are the best .
i want to give you a gift

**Comment by Lu-Yi-Hsun** [comment]
@Aleandro21  i seem fix it , some thing working

**Comment by Aleandro21** [comment]
@Lu-Yi-Hsun yes, but for me block first to generete a Chart.
how did you do ?
i receve always this error NameError: global name 'self__server_timestamp' is not defined

**Comment by Lu-Yi-Hsun** [comment]
@Aleandro21 

you can install last version  this api
that robot work on python2

**Comment by Aleandro21** [comment]
@Lu-Yi-Hsun thanks !!!!
